!!! TASK: Extract fluid dependencies into separate packages

mficzel opened the pull request:
The Fusion Prototype `Neos.Fusion:Template` is deprecated in favor of the new prototype `Neos.Fusion.FluidAdapter:Template`. 

All View Helpers and the dependencies are moved into the new packages `Neos.Fusion.FluidAdapter` and `Neos.Neos.FluidAdapter`.

**Upgrade instructions**

The prototype `Neos.Fusion:Template` you should change the prototype to `Neos.Fusion.FluidAdapter:Template` the original name will stay available until Neos 10.

**Review instructions**

This is based on https://github.com/neos/neos-development-collection/pull/4652 

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

mficzel committed:
TASK: Extract package `Neos.Fusion.FluidAdapter`

The Fusion Prototype `Neos.Fusion:Template` is deprecated together with the `fusion:render` view-helper.
Both are moved into the new package `Neos.Fusion.FluidAdapter`.

kitsunet commented:
Not opposed to deprecating, but maybe just move and note that the package will potentially be abandoned as a whole in the near future?

mficzel committed:
WIP !!! TASK: Extract package `Neos.Neos.FluidAdapter`

The view helpers of Neos.Neos are extracted into the new package `Neos.Neos.FluidAdapter` together with the fluid dependency.

mficzel commented:
Close and will repopen from private fork once ready
